<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authorAddingForm</class>
 <widget class="QWidget" name="authorAddingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add author</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="nameEdit">
     <property name="placeholderText">
      <string>Author name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Birth date</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="bioEdit">
     <property name="placeholderText">
      <string>Biography</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addBtn">
     <property name="cursor">
      <cursorShape>WhatsThisCursor</cursorShape>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
